dot_config/nvim/lua/tap/lsp/servers/tsserver.lua

```diff
@@ -49,7 +46,7 @@ function module.setup()
             end
         },
         cmd = vim.tbl_flatten({
```

Comment:
`tsserver` is using the wrong `typescript` version with this config, I'm guessing it's detecting the one installed by nvim-lsp-installer. It should prioritise the local version (and zero-install version) over local

Reply:
This looks to have come about through an update to the typescript-language-server - nvim-lspinstall setup uses the installed version too on the latest ts-ls version
https://github.com/typescript-language-server/typescript-language-server/pull/292

Reply:
Upstream issue so resolving for now